cli/src/config.rs

```diff
@@ -531,7 +531,7 @@ impl Config {
     }
 
     pub fn wallet_kp(&self) -> Result<Keypair> {
-        solana_sdk::signature::read_keypair_file(&self.provider.wallet.to_string())
+        get_keypair(&self.provider.wallet.to_string())
```

Comment:
Error message is still the same here. Need to remove the `map_err` line below.

Reply:
Ugh obviously 🤦🏼‍♂️ - thanks for your patience.